test/google_api_headers_test.dart

```diff
@@ -18,8 +20,8 @@ void main() {
     'google_api_headers',
   );
 
-  final platform = MockPlatform();
-  List<MethodCall> log;
+  final platform = MockMyPlatform();
+  late List<MethodCall> log;
```

Comment:
Can this be `late final`?

Reply:
Cannot, because `log` is initialized many times.

Reply:
You're right, I forgot it's initialized in every test, all good then